lib/lita/handlers/timdex_handler.rb

```diff
@@ -7,6 +7,8 @@ class TimdexHandler < Handler
 
       route(/^echo\s+(.+)/, :echo)
 
+      route(/^search\s+(.+)/, :search, help: { "search" => "Searches Timedex for the provided string of text"})
```

Comment:
Your auto correct is different than mine. I always get corrected to timex. Either way, the "official" name is TIMDEX! because it's more fun.